compiler/src/iree/compiler/GlobalOptimization/Passes.cpp

```diff
@@ -155,6 +155,8 @@ void buildGlobalOptimizationPassPipeline(
       });
 
   mainPassManager.addPass(DispatchCreation::createFoldUnitExtentDimsPass());
+  mainPassManager.addPass(
+      GlobalOptimization::createConvertStridedContractionToContractionPass());
```

Comment:
This pass location is fine. It should also work in the backend but either way is ok.